cmd/poller/plugin/plugin.go

```diff
@@ -184,7 +184,8 @@ func GetInterval(param *node.Node, defaultInterval time.Duration) float64 {
 }
 
 type DerivedMetric struct {
-	Name   string
-	Source string
-	IsMax  bool
+	Name       string
+	Source     string
+	IsMax      bool
+	CustomName bool
```

Comment:
Would `HasCustomName` or `HasCustom` be clearer?

Reply:
used `HasCustomName`